modules/volunteers/sitesupport/hourTable.js

```diff
@@ -110,6 +36,90 @@ export default {
     }
   },
   methods: {
+    load() {
+      this.records = null;
+      this.totalHours = 0;
+      this.totalSpentHours = 0;
+      this.hours = [];
+      this.totalVolunteers = null;
```

Comment:
Some of these values are already being set in the initial `data()` function - is there a reason why we need to set them again here?